docs/advanced-usage/connecting-cloud-storage/azure-blob-storage.md

```diff
@@ -0,0 +1,82 @@
+---
+icon: simple/microsoftazure
+---
+
+# Connecting Cloud Storage: <nobr>:simple-microsoftazure: Azure Blob Storage</nobr>
+
+Kolena connects with [Azure Blob Storage](https://azure.microsoft.com/en-ca/products/storage/blobs) to load files (e.g. images, videos, documents) directly
+into your browser for visualization. In this tutorial, we'll learn how to establish an integration between Kolena and
+Azure Blob Storage.
+
+To get started, ensure you have administrator access within Kolena.
+Navigate to the "Integrations" tab on the [:kolena-organization-16: Organization Settings](https://app.kolena.io/redirect/organization?tab=integrations)
+page and click "Add Integration", then "Azure Blob Storage".
+
+### Step 1: Create Azure App Registration for Kolena
+
+Azure Blob Storage integrations load resources using [shared access signatures](https://learn.microsoft.com/en-us/azure/storage/common/storage-sas-overview).
+Kolena generates these signatures using delegated access to these resources.
+We will generate an App registration for Kolena in Azure and then assign roles to this registration:
+
+1. From the [Azure portal](https://portal.azure.com/#home), search for "App registrations" and navigate to this page
+1. Click "New Registration"
+    1. Under "Supported account types", select "Accounts in any organizational directory"
+    1. Click "Register" to save the App registration
+1. Click on the App registration you have created
+1. Note the "Tenant ID" and "Application (client) ID"
+1. Click "Certificates & secrets", then "New client secret"
+    1. Click "Add" to save this secret and note the key value
+
+### Step 2: Assign Roles to App Registration
+
+We will assign two roles to the App registration created above:
+
+1. [Storage Blob Delegator](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#storage-blob-delegator) at the storage account level
+1. [Storage Blob Data Reader](https://learn.microsoft.com/en-us/azure/role-based-access-control/built-in-roles#storage-blob-data-reader) at the container level
+
+First we will assign the Storage Blob Delegator role:
```

Comment:
Should each of these numbered lists live under their own subheader? Slightly confusing to have stepped lists 1-2, 1-6, and 1-6 all under "Step 2".